lucene/core/src/java/org/apache/lucene/util/graph/GraphTokenStreamFiniteStrings.java

```diff
@@ -210,85 +199,41 @@ private void finish() {
    */
   private void finish(int maxDeterminizedStates) {
     Automaton automaton = builder.finish();
-
```

Comment:
So all this code here removed wasn't needed after all?  It's nice to see it all go away (less to maintain / less complexity) :-)